WalletWasabi.Fluent/ViewModels/MainViewModel.cs

```diff
@@ -101,7 +101,8 @@ public MainViewModel(Global global)
 
 			if (!_walletManagerViewModel.WalletManager.AnyWallet(_ => true))
 			{
-				MainScreen.To(_addWalletPage);
+				_addWalletPage.Navigate().To(_addWalletPage, NavigationMode.Clear);
+				// MainScreen.To(_addWalletPage);
```

Comment:
Can you remove this commented code?

Reply:
Yes I will remove, but first I need to finish investigation, as there is some potential issue I want to debug bit more.

Reply:
Remove commented code, I think it just is how navigation stack works it case you navigate from current target.